boa/src/builtins/object/mod.rs

```diff
@@ -410,6 +410,24 @@ pub fn make_object(_: &Value, args: &[Value], ctx: &mut Interpreter) -> ResultVa
     Ok(object)
 }
 
+/// Creates a new object from the provided prototype
+///
+/// https://tc39.es/ecma262/#sec-object.create
+/// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create
```

Comment:
we should change the documentation here to be like `Object.hasOwnProperty()` (`has_own_property`), I know some don't have it like `Object.is()` we should also update them, we can change it in this PR or in another one for `is`